### PR TITLE
Show post dates

### PR DESCRIPTION
**Pull request by annamvoitika**

### CODE REVIEW

**Review by kate-49 — COMMENTED** [review]
Can't see any obvious issues

File: cypress/integration/dates_spec.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('Timeline', function() {
+  it('can display dates', function() {
+    cy.visit('/posts');
+    cy.get('.posts').should('contain', 'British Summer Time');
```
Comment (twizlar12) [inline cypress/integration/dates_spec.js:10]: Should we add a step where we add a post? When we are able to have the DB clear before each integration test then this test might fail?

Comment (annamvoitika) [reply cypress/integration/dates_spec.js:10]: sure, can do

Comment (annamvoitika) [reply cypress/integration/dates_spec.js:10]: done :)

**Pull request merged**